src/lib.rs

```diff
@@ -140,15 +140,30 @@ pub async fn create_dir<P: AsRef<Path>>(path: P) -> io::Result<()> {
     unblock(move || std::fs::create_dir(path)).await
 }
 
-/// Creates a directory and its parent directories if they are missing.
+/// Creates a new, empty directory at the provided path
```

Comment:
Why was the old above description changed?

Reply:
Sorry about that, editing files through GitHub...